test/format/smoke.sh

```diff
@@ -16,6 +16,9 @@ args="$args runs.threads=4 "
 # $TEST_WRAPPER ./t $args runs.type=row runs.source=lsm
 # $TEST_WRAPPER ./t $args runs.type=var
 
-$TEST_WRAPPER ./t $args runs.type=row
-# Force a rebalance to occur with statistics logging to test the utility
-$TEST_WRAPPER ./t $args runs.type=row statistics.server=1 ops.rebalance=1
+# Run this loop for 10 times, short running stress jobs.
+for i in $(seq 10); do
```

Comment:
Is there a way to measure actual elapsed time? Sometimes a test/format run takes surprisingly long time. The 1 minute is for the core part of the program, but there are often other things added on that take material time.

Reply:
I've re-read your description, and I think I prefer using the `format.sh` `-t` argument, if that's OK with you?

Reply:
@agorrod Thanks for reviewing the changes. Yes, Even I prefer to use `format.sh` the script which manages the time internally.